src/api/api.js

```diff
@@ -0,0 +1,62 @@
+export const API_SERVER = 'https://kdt-frontend.programmers.co.kr/documents';
+
+export const request = async (url = '', options = {}) => {
+  try {
+    const res = await fetch(`${API_SERVER}${url}`, {
+      ...options,
+      headers: {
+        'x-username': 'juyeon',
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (res.ok) {
+      return await res.json();
+    }
+
+    throw new Error('API 처리중 오류 발생');
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const getAllDocuments = async () => {
+  const documents = await request('');
+  return documents;
+};
+
+export const getDocument = async id => {
+  const document = await request(`/${id}`);
+  return document;
+};
+
+export const createDocument = async (title, parentId) => {
+  const res = request('', {
+    method: 'POST',
+    body: JSON.stringify({
+      title,
+      parent: parentId,
+    }),
+  });
```

Comment:
```suggestion
export const createDocument = async (title, parent) => {
  const res = request('', {
    method: 'POST',
    body: JSON.stringify({
      title,
      parent,
    }),
  });
```

음... `parentId` -> `parent`로만 바꿔도 **body** 영역의 라인을 줄일 수 있을것 같아서 더 효과적이지않을까 생각되네요 👍